cirq/study/sweepable.py

```diff
@@ -75,7 +77,9 @@ def to_sweeps(sweepable: Sweepable) -> List[Sweep]:
     if isinstance(sweepable, Iterable) and not isinstance(sweepable, str):
         return [
             sweep for item in sweepable for sweep in to_sweeps(
-                cast(Union[Dict[str, float], ParamResolver, Sweep], item))
+                cast(
```

Comment:
This is also quite unwieldy.  Should we alias this union type?

Reply:
Fixed. Should have been `Sweepable`.